bin/generate.js

```diff
@@ -47,6 +48,19 @@ mkdirp(outputPath, err => {
     fs.copyFileSync(logoPath, path.join(outputPath, 'logo.png'));
   }
 
+  try {
+    const data = fs.readFileSync(path.join(outputPath, 'index.html'), 'utf8')
+    var result = data.replace('DATAROOT', dataRoot);
+  } catch (err) {
+    console.error(err)
+  }
+
+  try {
+    fs.writeFileSync(path.join(outputPath, 'index.html'), result)
```

Comment:
Any reason this is in a separate try-catch block? I feel like this operation is minuscule enough to include in the previous block (since both possible errors are related to file I/O, I think it'd be good to encapsulate them within the same scope).